### PR TITLE
Pipeline summary wrap

### PR DESCRIPTION
**Pull request by gildub**
Resolves https://github.com/konveyor/forklift-ui/issues/283

### CODE REVIEW

**Comment by gildub** [comment]
@vconzola, please check it out and let me know if that works for you.

File: src/app/common/components/PipelineSummary.tsx [hunk 1]
```diff
@@ -35,11 +35,8 @@ export const getPipelineSummaryTitle = (status: IVMStatus): string => {
     return 'Complete';
   }
   if (status.started && !status.completed) {
-    let title = status.error ? 'Error' : 'Running';
-    if (currentStep?.description) {
-      title = `${title} - ${currentStep.description}`;
-    }
-    return title;
+    if (status.error) return `error - ${currentStep?.description}`;
```
Comment (mturley) [inline src/app/common/components/PipelineSummary.tsx:38]: You lost the capital E on "error" here

**Comment by mturley** [comment]
@gildub I'm seeing 2 problems with this:
* The tooltip appears all the time, even when all the text is visible
* There are no ellipses, the text just gets cut off by the edge of the container

I think you can probably get rid of the `cellTransforms: [truncate]` and instead use [our custom TruncatedText component](https://github.com/konveyor/forklift-ui/blob/master/src/app/common/components/TruncatedText.tsx) around the text instead of using a Tooltip directly.

**Comment by vconzola** [comment]
@gildub A few minor things... (1) "error" should be capitalized. (2) Are errors always fatal - meaning the migration has failed and stopped?  If so, the icon in the pipeline should be a full red circle, not a partially full one. (3) We should be consistent with the use of periods after the step names. I prefer always having NO periods.

**Comment by mturley** [comment]
Interesting point -- I think you're right @vconzola, but I had been looking at those partially-full red circles and thinking of them as "this step failed while in progress" vs a full circle which could mean "this step ran all the way through and resulted in a failure". I don't think we really have a distinction there though.

**Comment by vconzola** [comment]
@mturley I think distinguishing between this step failed while in progress and this step caused the migration to fail is too nuanced to worry about. If an error causes a migration to stop, I'd like to use the full, red icon.

**Comment by mturley** [comment]
Makes sense.

**Comment by gildub** [comment]
@mturley, I initially tried with TruncatedText component but I wasn't happy with the result, I think I know why so let me give it another go.

@vconzola, no worries to change to display a full red circle icon when an error occurred.

I've restored the capital on "error".

**Comment by mturley** [comment]
Sounds good. Let me know if you need any help, the TruncatedText component is a little weird (CSS `width: 100%` works differently depending on the container it's in).

**Review by mturley — APPROVED** [review]
Looks good to me. I can't actually get it to truncate with the mocked text unless I make my window very small, but it looks like it would work correctly at normal sizes with longer messages, so this is perfect.

@vconzola can you give it a review? As far as periods in the names, that's coming straight from the backend, so we should open an issue with Jeff to fix that unless we want to filter out all trailing periods in the UI code.

**Comment by vconzola** [comment]
@gildub @mturley LGTM. I'll open an issue for Jeff. What component would that be?

**Comment by mturley** [comment]
@vconzola virt-controller I think. Or I guess it's forklift-controller now.

**Pull request merged**